web/README.md

```diff
@@ -0,0 +1,418 @@
+# HarbourBridge: Web APIs (Experimental)
+
+HarbourBridge is a stand-alone open source tool for Cloud Spanner evaluation,
```

Comment:
We're evolving HarbourBridge beyond just evaluation. Suggest changing this to:

HarbourBridge is a stand-alone open source tool for Cloud Spanner evaluation and migration.

and dropping "using data from an existing PostgreSQL or MySQL database." since that's not relevant to this README.

Reply:
updated statement